.github/workflows/ci.yml

```diff
@@ -12,16 +12,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Install Ruby
-        uses: eregon/use-ruby-action@v1.10.0
+        uses: eregon/use-ruby-action@v1
         with:
           ruby-version: ${{ matrix.ruby }}
       - name: Install dependencies
         run: |
-          gem install -N bundler
-          # NOTE using bundle config causes bundle install to crash; see https://github.com/eregon/use-ruby-action/issues/13
-          #bundle config --local path .bundle/gems
-          #bundle config --local without docs lint coverage
-          #bundle --jobs 3 --retry 3
-          bundle --jobs 3 --retry 3 --path .bundle/gems --without docs lint coverage
+          bundle config --local path .bundle/gems
+          bundle config --local without docs lint coverage
+          # warning output can confuse Powershell in CI, (multi-line scripts & exit codes)
+          $env:RUBYOPT = '-W0'
```

Comment:
1. What if bundler call is split into a separate step? Does it also fail to print warnings?
2. You lost `--without docs lint coverage`

Reply:
@mojavelinux and what's the point storing gems in `.bundle/gems`? Ain't everything supposed to work regardles of gems location? Whole CI VM will be dropped anyway after build is finished.

Reply:
@slonopotamus 

> What if bundler call is split into a separate step? Does it also fail to print warnings

It's not about it failing to print warnings.  It's about warnings causing the step to fail when they shouldn't.

There's been discussion in RubyGems about this, in that warnings are printed when a user has no control over the gem in question.  The idea was to change the warnings to only show when the gem is built, not when it is installed.

Reply:
The use of path is no about caching. It's about testing the installation of the CLI correctly and in isolation. Without specifying a path, the CLI location test has to be partially faked.